cmd/bbolt/main.go

```diff
@@ -1050,9 +1051,23 @@ func (cmd *benchCommand) Run(args ...string) error {
 	}
 
 	// Print results.
-	fmt.Fprintf(cmd.Stderr, "# Write\t%v(ops)\t%v\t(%v/op)\t(%v op/sec)\n", writeResults.CompletedOps(), writeResults.Duration(), writeResults.OpDuration(), writeResults.OpsPerSecond())
-	fmt.Fprintf(cmd.Stderr, "# Read\t%v(ops)\t%v\t(%v/op)\t(%v op/sec)\n", readResults.CompletedOps(), readResults.Duration(), readResults.OpDuration(), readResults.OpsPerSecond())
+	if options.GoBenchOutput {
+		// below replicates the output of testing.B benchmarks, e.g. for external tooling
+		maxLen := 14 // the length of "BenchmarkWrite"
+		gobenchResult := testing.BenchmarkResult{}
+		gobenchResult.T = writeResults.Duration()
+		gobenchResult.N = int(writeResults.CompletedOps())
+		fmt.Fprintf(cmd.Stderr, "%-*s\t%s\n", maxLen, "BenchmarkWrite", gobenchResult.String())
```

Comment:
Should we (would it be possible to) print this to stdout rather than stderr? I think that it would be cleaner for other tools to redirect stdout and stderr accordingly to use its output rather than using `grep`.

Reply:
I'd be +1 for that, just left it consistent with the existing output

Reply:
This is a generic issue. I think it's OK to resolve it in a separate PR.